services/backend/API/admin/post.js

```diff
@@ -25,13 +25,26 @@ const bulkDeleteOption = async (request, response) => {
     const ids = request.body.ids;
     const model = getModel(type);
 
-    model
+    let errorCaught = false;
+    let result;
+
+    await model
       .destroy({
-        where: { id: ids }
+        where: { id: ids },
+      })
+      .then((destroyCount) => {
+        result = destroyCount > 0;
       })
-      .then(destroyCount =>
-        response.status(200).json({ deletePerformed: destroyCount > 1 })
-      );
+      .catch(function () {
+        console.log("Delete Error!");
+        errorCaught = true;
+      });
+
+    if (errorCaught === true) {
+      result = false;
```

Comment:
You should be able to put result = false into line 40

Reply:
Fixed!